tests/testthat/test-kaldi.R

```diff
@@ -1,32 +1,37 @@
 test_that("kaldi__get_lr_indices_and_weights", {
-  expect_no_error(x <- kaldi__get_lr_indices_and_weights(
-  orig_freq = 4,
-  new_freq = 3,
-  output_samples_in_unit = 6,
-  window_width = 2,
-  lowpass_cutoff = 1,
-  lowpass_filter_width = 2,
-  device = torch::torch_device("cpu"),
-  dtype = torch::torch_float()
-  ))
-  expect_tensor(x[[1]])
-  expect_tensor(x[[2]])
-  expect_error(x[[3]])
+  if(requireNamespace("numbers", quietly = TRUE)) {
```

Comment:
We usually use `testthat::skip_if_not_installed()` in that case to not completely silently avoid the test.

Reply:
Oh sure, thanks!